libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/internal/factory/DropInUIMapboxNavigationFactory.kt

```diff
@@ -0,0 +1,24 @@
+package com.mapbox.navigation.dropin.internal.factory
+
+import android.content.Context
+import com.mapbox.navigation.base.options.NavigationOptions
+import com.mapbox.navigation.core.MapboxNavigation
+import com.mapbox.navigation.core.MapboxNavigationProvider
+
+class DropInUIMapboxNavigationFactory(
```

Comment:
I don't think calling it `MapboxNavigationFactory` is quite appropriate since we don't want external developers to use it.  It should be called something that doesn't imply it's available for use. Since we don't use the term "drop in" outside of ourselves it doesn't encourage external usage since the term has no meaning outside of our group. I'm open to other suggestions.

Reply:
I think it's more of a developer preference but I would rather do the following:
- Remove internal package
- Keep the factory package
- Mark all files in the factory internal (Doing this we can have the file as `internal class MapboxNavigationFactory`) thereby making it unaccessible to the developers.) Same applies to `MapboxNavigationViewModelFactory`
- Mark all the view models we create as internal (This is a must and should happen to both `MapboxNavigationViewModel` and `CameraViewModel`)